public/index.html

```diff
@@ -1,138 +1,320 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" href="/favicon.ico" type="image/x-icon">
-  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
-  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
-  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
-  <link rel="manifest" href="/site.webmanifest">
-  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#3531ff">
-  <meta name="msapplication-TileColor" content="#2b5797">
-  <meta name="theme-color" content="#ffffff">
-  <meta name="description" content="State channels are a scalability technology for ethereum enabling instant & free peer-to-peer ethereum transactions">
-  
-  <meta property="og:description" content="State channels are a scalability technology for ethereum enabling instant & free peer-to-peer ethereum transactions">
-  <meta property="og:site_name" content="State Channels - Instant P2P Ethereum Transactions">
-  <meta property="og:title" content="State Channels - Instant P2P Ethereum Transactions">
-  <meta property="og:url" content="https://statechannels.org/">
-  <meta property="og:type" content="website">
-  <meta property="og:image" content="https://statechannels.org/assets/symbol.png">
-  <meta property="og:image:width" content="400" />
-  <meta property="og:image:height" content="400" />
-  
-  <meta name="twitter:title" content="State Channels - Instant P2P Ethereum Transactions">
-  <meta name="twitter:description" content="State channels are a scalability technology for ethereum enabling instant & free peer-to-peer ethereum transactions">
-  <meta name="twitter:card" content="summary">
-  <meta name="twitter:url" content="https://statechannels.org/">
-  <meta name="twitter:image" content="https://statechannels.org/assets/symbol.png">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
It's unfortunate that this commit includes format changes as that makes the diff quite large.

Reply:
Apologies. I've made another PR to add `prettier` so we don't get that on future PRs 🙏 .

Reply:
#53